pages/reports.js

```diff
@@ -0,0 +1,210 @@
+import s from 'styled-components'
+import {useState} from 'react'
+import renderPage from '../renderPage.js'
+import { Icon, Flex } from '../components/common'
+import {API_BASE_URL} from '../utils'
+import { Sidebar } from '../components/common/Sidebar'
+import Checkbox from '../components/common/Checkbox'
+import { Container, WideContainer } from '../components/common/Container'
+import { CLUBS_GREY, RED, ALLBIRDS_GRAY, MEDIUM_GRAY } from '../constants/colors'
+
+
+const TallTextArea = s.textarea`
+  height: 6em;
+`
+
+const GroupLabel = s.h4`
+  margin-top: 1em;
+  margin-bottom: 0em !important;
+  font-size: 32px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.8px;
+  color: #626572;
+`
+
+const HoverListElement = s.li`
+
+  span {
+    display: none;
+  }
+
+  &:hover span {
+    display: inline;
+    cursor: pointer;
+  }
+
+`
+
+const SelectedManager = ({ value, onClick }) => (
+  <HoverListElement>
+    {value}
+    
+    <span style={{marginLeft: '1em'}} onClick={onClick}>X</span>
+  </HoverListElement>
+)
+
+const Reports = () => {
+  const fields = {  // TODO: Get this from the server.
+    'Basics': [
+      'Name',
+      'Subtitle',
+      'Description',
+      'Date Founded',
+      'Size',
+      'Activity',
+      'Approval Status',
+      'Parent Orgs',
+      'How to Get Involved',
+      'Tags'
+    ],
+    'Additional Links': [
+      'Website',
+      'Facebook',
+      'Instagram',
+      'Twitter',
+      'Github',
+      'Linkedin'
+    ],
+    'Contact Information': [
+      'Owner name(s)',
+      'Owner email(s)',
+      'Listserv',
+      'Officer name(s)',
+      'Officer email(s)'
+    ]
+  }
+
+  const nameToCode = {
```

Comment:
we should probably modularize this more. Have a folder for components relating to `reports` then we can have a `constants.js` file where we import these things. IMO having them in the file doesn't improve readability and increases the amount ppl have to scroll.

Reply:
Also this constant is presumably useful / already used elsewhere? not sure

Reply:
This has been (partially) changed to be loaded from the API server, so that when we update backend fields we don't have to remember to update frontend too.

The groups of fields constant has not been changed yet.